tests/test_pysnooper.py

```diff
@@ -1172,3 +1173,21 @@ def sum_to_x(x):
             ReturnValueEntry('49995000'),
         )
     )
+
+
+def test_activate_deactivate_snoop():
+    string_io = io.StringIO()
+
+    def my_function(foo):
+        x = 7
+        y = 8
+        return x + y
+
+    os.environ['PYSNOOPER_DISABLED'] = '1'
+    with pysnooper.snoop():
```

Comment:
```suggestion
    with pysnooper.snoop(output):
```

Right now you're not actually testing anything.

Reply:
yes! I'm wrong. it's should pass the `string_io` object. I'll change it.